Makefile

```diff
@@ -445,10 +445,11 @@ ifeq ($(CI), true)
 	gocov convert $(REPORTS)/$(TEST_SCENARIO)_coverage.out | gocov-xml > $(REPORTS)/$(TEST_SCENARIO)_coverage.xml
 endif
 
+# The image quay.io/edge-infrastructure/postgresql-12-centos7:latest is built from https://github.com/sclorg/postgresql-container
```

Comment:
I wonder if there is somewhere better we could put this? Do we have some kind of testing or dev docs?

Reply:
Maybe we can add here `assisted-service/docs/dev/testing.md`